dpppt-config-backend/src/main/java/org/dpppt/switzerland/backend/sdk/config/ws/model/ConfigResponse.java

```diff
@@ -107,4 +110,12 @@ public void setInterOpsCountries(List<String> interOpsCountries) {
 	public void setTestInformationUrls(Map<String, String> testInformationUrls) {
 		this.testInformationUrls = testInformationUrls;
 	}
+
+	public void setCheckInUpdateNotificationEnabled(boolean checkInUpdateNotificationEnabled) {
+		this.checkInUpdateNotificationEnabled = checkInUpdateNotificationEnabled;
+	}
+
+	public boolean isCheckInUpdateNotificationEnabled() {
```

Comment:
Is the `isCheckInUpdateNotificationEnabled` method necessary? It doesn't look like it's used anywhere